includes/frontend/class-wcmp-cart-fees.php

```diff
@@ -250,6 +257,13 @@ private function getFeeTitles(): array
      */
     private function addDeliveryFee(): void
     {
-        $this->addFee("delivery_{$this->deliveryOptions->getDeliveryType()}");
+        $deliveryType = $this->deliveryOptions->getDeliveryType();
+
+        if ($this->deliveryOptions->getShipmentOptions()
+            && $this->deliveryOptions->getShipmentOptions()->isSameDayDelivery()) {
+            $deliveryType = 'same_day';
```

Comment:
misschien toch even een constante hiervan maken, en dan bij die constante nog een keer toelichten dat het eigenlijk een shipment option is maar wij doen alsof het een delivery type is...